spec/lib/roo/google_spec.rb

```diff
@@ -75,9 +75,9 @@ def key_of(spreadsheetname)
     end
     let(:key) { key_of('numbers1') }
     it 'check date fields' do
-      expect(subject.celltype(5,1)).to eq(:date)
-      expect(subject.cell(5,1)).to eq(Date.new(1961, 11, 21))
-      expect(subject.cell(5,1).to_s).to eq('1961-11-21')
+      expect(subject.celltype(5,'A')).to eq(:date)
+      expect(subject.cell(5,'A')).to eq(Date.new(1961, 11, 21))
+      expect(subject.cell(5,'A').to_s).to eq('1961-11-21')
```

Comment:
Is the above changeset necessary?

Reply:
No probably not, as I was experimenting. It is changed back